src/libraries/System.Private.CoreLib/src/System/Numerics/Vector4.cs

```diff
@@ -183,30 +197,33 @@ public static Vector4 UnitW
             );
         }
 
-        /// <summary>Multiplies a vector by the given scalar.</summary>
-        /// <param name="left">The source vector.</param>
+        /// <summary>Multiples the specified vector by the specified scalar value.</summary>
```

Comment:
Typo: Should be "Multiplies"

Reply:
Thanks for catching those. I think fixing them by hand is the best approach here. This wasn't a problem with the tool, but rather a real typo in the source.